docs/src/testing/the-setup-program-test-macro.md

```diff
@@ -31,7 +31,7 @@ Example: `Wallets("a_wallet", "another_wallet"...)`
 
 Description: Launches a local provider and generates wallets with names taken from the provided `ARG`s.
 
-Cardinality: 0 or 1.
+Cardinality: 0 or N.
```

Comment:
Actually, it is 0 or 1. Having multiple `Wallets` commands is an error.

Reply:
You are correct. I thought it refers to the wallet names and not the command itself. I will close the PR